package.json

```diff
@@ -34,7 +34,6 @@
   ],
   "main": "./dist/extension",
   "scripts": {
-    "preinstall": "npx npm-force-resolutions",
```

Comment:
why removing it?

Reply:
The `package-lock.json` format changed in npm 7, so it npm-force-resolutions doesn't work correctly: https://stackoverflow.com/a/65014694.

I shouldn't have made this change, since this likely will cause issues on Jenkins. I'll downgrade my npm so that it will work on my machine.